### DISCUSSION TITLE
Support for node

### DISCUSSION THREAD

**Discussion by leo-paz** [Unstructured-IO/unstructured#1592] [General]
Hey Unstructured Team,

Love what you guys are doing and love that you guys made it so easy to self host but I was wondering if there was any plans for having a node sdk?

**Comment by cragwolfe** [comment]
We are planning an SDK javascript client very soon, for an improved experience in interacting with the API. Hope this helps!

**Reply by leo-paz** [reply]
Awesome :) it would be great if it was compatible with edge runtimes out of the box (eg. fetch over axios)

**Comment by selimrbd** [comment]
Hello there !

Any update / expected timeline for the release of the JS SDK ?
It would be great to interact with your local API as easily as in python
